source/AI.cpp

```diff
@@ -1773,16 +1775,17 @@ void AI::DoScatter(Ship &ship, Command &command)
 
 
 
-Point AI::StoppingPoint(const Ship &ship, bool &shouldReverse)
+// Instead of coming to a full stop, adjust to a target velocity vector
+Point AI::AdjustPoint(const Ship &ship, const Point &targetVelocity, bool &shouldReverse)
 {
 	const Point &position = ship.Position();
-	const Point &velocity = ship.Velocity();
+	const Point &velocity = ship.Velocity() - targetVelocity;
```

Comment:
It doesnt flag it, it compiles without any warnings or alike.
As I wrote a few days ago, I am new to C++, lacking experience in objects and classes, and thus can make mistakes like these. And I am not even a native english speaker, to add on top. :-(
Would you please (simply) fix it for me?